.travis.yml

```diff
@@ -157,6 +157,6 @@ jobs:
         FILE_ENV="./ci/test/00_setup_env_amd64_nowallet.sh"
 
     - stage: test
-      name: 'macOS 10.10  [GOAL: deploy] [no functional tests]'
+      name: 'macOS 10.12  [GOAL: deploy] [no functional tests]'
```

Comment:
So the macOS version is `10.12`, the SDK version is `10.14`, and the cross compile target is `x86_64-apple-darwin16`? This is not confusing at all.

Reply:
`x86_64-apple-darwin16` maps to macOS `10.12`, more info [here](https://en.wikipedia.org/wiki/Darwin_%28operating_system%29#Release_history). We're building with the 10.14 SDK, however this can be newer, and is the same as we've been doing in previous releases.